projects/packages/sync/src/modules/class-posts.php

```diff
@@ -618,12 +618,10 @@ public function save_published( $new_status, $old_status, $post ) {
 	 * The 2nd request is to update post meta, which is not supported on WP REST API.
 	 * When syncing post data, we will include if this was a meta box update.
 	 *
-	 * @todo Implement nonce verification.
-	 *
 	 * @return boolean Whether this is a Gutenberg meta box update.
 	 */
-	public function is_gutenberg_meta_box_update() {
-		// phpcs:disable WordPress.Security.NonceVerification.Missing, WordPress.Security.NonceVerification.Recommended
+	private function is_gutenberg_meta_box_update() {
```

Comment:
From a semver perspective, changing this from public to private needs a major version bump. Unless we consider this function to never have been part of the package's public API?

Reply:
Yes, there's absolutely no reason for this method to be part of the package's public API. It only makes sense as a private method in the `Posts` class and I can't imagine any case where it would be valuable to any consumer under the `Posts` Sync module context.